snippets/javascript/statements.md

```diff
@@ -0,0 +1,50 @@
+# JS Statements
+
+## If statement
+
+```js
+const isMailSent = true;
+
+if (isMailSent) {
+  console.log("Mail sent to recipient");
+}
+```
+
+<!-->An if statement accepts an expression with a set of parentheses:
+
+If the expression evaluates to a truthy value, then the code within its code
+body executes. If the expression evaluates to a falsy value, its code body will
+not execute.<!-->
+
+## Else Statement
+
+```js
+const isTaskCompleted = false;
+
+if (isTaskCompleted) {
+  console.log("Task completed");
+} else {
+  console.log("Task incomplete");
+}
+```
+
+<!-->An else block can be added to an if block or series of if-else if blocks. The else block will be executed only if the if condition fails.<!-->
+
+## Else if Statement
+
+```js
+const size = 10;
+
+if (size > 100) {
+  console.log("Big");
+} else if (size > 20) {
+  console.log("Medium");
+} else if (size > 4) {
+  console.log("Small");
+} else {
+  console.log("Tiny");
+}
+// Print: Small//
```

Comment:
copied by accident?

Reply:
Hi @arnochauveau Not sure what you mean by copied by accident?

Reply:
The `// Print: Small //` in your code? 😄 I think it was copied by accident.

Reply:
It means that the this code will print Small to the console I thought, maybe that is wrong

Reply:
It's a comment, so it would surprise me if it did. 

You can style the log result with css (example font-size) by using `console.log('%c Your text', 'font-size: 10px; other-css-properties: value')`. This works for chrome, not sure about other browsers.

Anyway, the snippets should be focused on the topic of the snippet as all the extra stuff might be confusing for others 🙂

Reply:
Ok no worries I will remove it